### PR TITLE
Add instructions to configure service account in README

### PR DESCRIPTION
**Pull request by etabakov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ with:
   debugSymbols: app/intermediates/merged_native_libs/release/out/lib
 ```
 
+## Configure service account
+1. Configure service account in Google Cloud Platform
+   1. Navigate to https://cloud.google.com/gcp
+   1. Open IAM and admin > Service accounts > Create service account
```
Comment (boswelja) [inline README.md:54]: Should these all have different numbers? They appear to all be `1.`

**Review by boswelja — APPROVED** [review]
Looks good, aside from the small nit I commented about

File: README.md (hunk 1 above)
Comment (etabakov) [reply README.md:54]: This is how markdown syntax works. If you jump to preview, you will see that items are properly numbered. The idea is the list to be easier for maintenance when you add, remove, or reorder items.

Comment (etabakov) [reply README.md:54]: Here is the preview: https://github.com/r0adkll/upload-google-play/blob/76b2dab280a4ed623c5d9006f52cfc579aa32736/README.md

Comment (boswelja) [reply README.md:54]: I did check the preview, I've just never seen that used before 😅

**Pull request merged**